ddtrace/contrib/internal/mcp/patch.py

```diff
@@ -268,11 +265,16 @@ def traced_request_responder_exit(mcp, pin: Pin, func, instance, args: tuple, kw
 async def traced_request_responder_respond(mcp, pin: Pin, func, instance, args: tuple, kwargs: dict):
     integration: MCPIntegration = mcp._datadog_integration
     span: Optional[Span] = getattr(instance, "_dd_span", None)
+
+    result = await func(*args, **kwargs)
+
     if span:
+        response_arg = args[0] if len(args) > 0 else None
```

Comment:
just to clarify (and if so a comment here might help) - does `response_arg`/`args[0]` get populated _after_ the function happens? otherwise, i think it's unclear to me why we await the function first